indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -268,6 +272,11 @@ protected void innerMap(
           )
       ).asBytes();
 
+      byte[] serializedInputRow = inputRow instanceof SegmentInputRow ?
```

Comment:
Are there any tests to make sure that delta ingestion of metrics with different regular/combining aggregators works?